packages/sdk/src/hooks/useCredentials.tsx

```diff
@@ -1,30 +1,70 @@
 import { useMemo } from 'react'
+import { type CredentialForDisplay, getCredentialForDisplay } from '../display/credential'
+import type { CredentialCategoryMetadata } from '../metadata/credentials'
 import { useMdocRecords } from '../providers/MdocProvider'
 import { useSdJwtVcRecords } from '../providers/SdJwtVcProvider'
 import { useW3cCredentialRecords } from '../providers/W3cCredentialsProvider'
 
-/**
- *
- * @todo add Display information to the records
- *
- */
 export const useCredentials = ({
   removeCanonicalRecords = true,
+  credentialCategory,
 }: {
   removeCanonicalRecords?: boolean
+  // TODO: remove this?
+  credentialCategory?: CredentialCategoryMetadata['credentialCategory']
```

Comment:
Why remove it? I think we're using this to retrieve the PID record

Reply:
Yeah mainly because we only really use it for the PID. Just wondering about how relevant that is in a general SDK.